modifiers_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright Axis Communications AB.
+//
+// For a full list of individual contributors, please see the commit history.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package eiffelevents
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func ExampleModifier() {
```

Comment:
Is this used?

Reply:
Yes! See https://pkg.go.dev/testing#hdr-Examples and https://go.dev/blog/examples.

I've been tinkering with the idea of writing a small program that extracts the examples from README.md to separate example files (see _Larger examples_ section in the blog article above). That would guarantee that the examples actually compile and work as expected, plus the pkg.go.dev page would include the examples separately in a playground-like manner; see https://pkg.go.dev/sort for an example. WDYT?

Reply:
Huh, that sounds pretty neat :)